.readthedocs.yaml

```diff
@@ -0,0 +1,23 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+
+# Build documentation in the "docs/" directory with Sphinx
+sphinx:
+  configuration: docs/source/conf.py
+
+# Optional but recommended, declare the Python requirements required
+# to build your documentation
+# See https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html
+python:
+  install:
+    - requirements: docs/requirements.txt
+    - requirements: requirements.txt
```

Comment:
Added this include so the package dependencies are installed, and mostly for the API docs - without it, most of the pages would not render as they would not be able to `import torch`, etc. The dependencies were removed from `doc/requirements.txt` at #456 - hopefully this will accomplish the same goals.